src/components/compress/index.tsx

```diff
@@ -568,6 +572,7 @@ export default class Compress extends Component<Props, State> {
 
     const options = sides.map((side, index) => (
       <Options
+        key={index}
```

Comment:
This seems unrelated to this PR

Reply:
Ah, just saw you mentioned this in the original comment. @developit does `key` make sense here? (and other places in the PR)

Reply:
Index keys are never required and often cause performance degradation. `side` could be used as a key here, though there is no reason to do so.